llpc/lower/llpcSpirvLowerRayTracing.cpp

```diff
@@ -347,6 +357,7 @@ void SpirvLowerRayTracing::visitReportHitOp(ReportHitOp &inst) {
     Value *paramHitKind = func->arg_begin() + 1;
     Value *bufferDesc = func->arg_begin() + 2;
     Value *shaderRecordIndex = func->arg_begin() + 3;
+    Function::arg_iterator traceIt = func->arg_begin() + nonTraceArgs;
```

Comment:
traceIt -> traceParams

Reply:
some place i changed to the traceParamsIt , because traceParams is used.